### PR TITLE
Implemented doctrine data fixtures

### PR DESCRIPTION
**Pull request by marioradu05**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,15 +141,7 @@ Note: you need to whitelist `localhost` in the reCAPTCHA settings page during de
...
-Run the migrations and seeds with these commands:
-
-```bash
-php vendor/bin/phinx migrate --configuration="config/migrations.php"
-php vendor/bin/phinx seed:run --configuration="config/migrations.php"
-```
-
-### Migration alternative with Doctrine
-Out of the box, we use Phinx like detailed above to populate the database. Doctrine is an alternative that is also ready to use for new migrations. An example file is included in `/data/doctrine/migrations`, but it's empty, so it won't run any queries. It can be edited freely. To generate a new migration file, use this command 
+Out of the box, we use Doctrine Migrations like detailed below to populate the database. An example file is included in `/data/doctrine/migrations`, but it's empty, so it won't run any queries. It can be edited freely. To generate a new migration file, use this command:
```
Comment (alexmerlin) [inline README.md]: Am I looking at the wrong file or `data/doctrine/migrations/Version20220817121035.php` is not empty, as claimed here?

File: README.md [hunk 2]
```diff
@@ -192,6 +184,20 @@ php vendor/bin/doctrine-migrations migrations:execute --down 20220606131835
 This will also remove the log for that migration in the database, allowing the migration to run again with `php vendor/bin/doctrine-migrations migrate`. 
 Note the `20220606131835` is taken from the migration filename, e.g. `Version20220606131835.php`
 
+## Seeding the database (Fixtures)
+Seeding the database uses doctrine data fixtures and our custom CLI implementation that sits on top of ``doctrine/data-fixtures`` and provides an interface for listing and executing fixtures.
```
Comment (alexmerlin) [inline README.md]: Would rephrase this to:

> Seeding the database is done with the help of our custom package [dotkernel/dot-data-fixtures](https://github.com/dotkernel/dot-data-fixtures) built on top of doctrine/data-fixtures. See below on how to use our CLI command for listing and executing Doctrine data fixtures.

File: README.md (hunk 1 above)
Comment (marioradu05) [reply README.md]: You are right, I've added the default migration class and edited the documentation.

**Review by alexmerlin — APPROVED** [review]
